### ISSUE TITLE
Omit the "Controller" suffix in the controller `<name>`?

### ISSUE THREAD

**Issue by dvlprs** [cakephp/bake#287]
[Controller conventions](http://book.cakephp.org/3.0/en/intro/conventions.html#controller-conventions) suggest using class names that are *plural, CamelCased, and end in `Controller`*. 

So I tried to bake an `ArticlesController`, which resulted in `src/Controller/ArticlesControllerController.php`.

Is that expected behavior, or do you think the bake console should auto remove (or suggest to remove) the `Controller` part?

**Comment by markstory** [comment]
For each object type Bake always expects the name without the suffix. I'm hesitant to remove words from the input, as I'm certain someone will need to create something `DinersTablesTable` eventually.

Would updating the `--help` of each command to be clearer help?

**Comment by dvlprs** [comment]
I did read `--help` before running the command, however I haven't noticed the *Arguments* section until I've taken a closer look after reading your response. So if you place this info into it, there's a chance some people won't notice it anyway -- just because it's not that obvious. 

I understand automatically altering user's input might be not obvious either, and confusing too (however, CakePHP already relies heavily on automagic based on conventions, and it seems to work fine). That's why I said *or suggest to remove*, i.e. prompt an `Are you sure you wanted to bake ArticlesControllerController? Choose n for ArticlesController [Y/n]` question before continuing. 

I don't have any data on how many people ran into this issue, and what's the overall preference. So probably you're right, altering `--help` might be enough for now, and whoever has some more input, can find this issue and comment (I searched and couldn't find anything relevant, so created a new issue).
